ArcGISRuntimeSDKQt_CppSamples/EditData/EditFeatureAttachments/EditFeatureAttachments.cpp

```diff
@@ -237,12 +237,13 @@ void EditFeatureAttachments::onQueryFeaturesCompleted_(FeatureQueryResult* featu
     emit attachmentModelChanged();
 
     // get the number of attachments
-    connect(m_selectedFeature->attachments(), &AttachmentListModel::fetchAttachmentsCompleted,
-            this, [this](const QUuid&, const QList<Attachment*>& attachments)
-    {
-      m_mapView->calloutData()->setDetail(QString("Number of attachments: %1").arg(attachments.size()));
-      m_mapView->calloutData()->setVisible(true); // Resizes the calloutData after details has been set.
-    });
+    // enableAutoFetch and enableAutoApplyEdits arguments of attachments() is set as false to avoid auto-fetching and auto-apply edits.
```

Comment:
I would tweak this
```suggestion
    // enableAutoFetch and enableAutoApplyEdits for AttachmentListModel are set as false to avoid automatic behavior. We will call fetchDataAsync explicitly as needed.
```